app/assets/js/components/Work/CardItem.jsx

```diff
@@ -29,7 +29,7 @@ const WorkCardItem = ({
         <figure className="image is-3by3">
           <Link to={`/work/${id}`}>
             <UIWorkImage
-              imageUrl={getIIIFImageUrl(representativeImage)}
```

Comment:
It looks like this PR is removing the only two references to this function, so it might make sense to remove the imports in both files and also the function itself from `js/services/helpers.js`

Reply:
Done